wmcontrol.py

```diff
@@ -483,14 +489,18 @@ def loop_and_submit(cfg):
                       print "no white listing"
                   if test_mode:
                       print "Finished in", int((time.time()-t)*1000), "\bms"
-
           if params['RunWhitelist']:
               params['Task1']['RunWhitelist'] = params['RunWhitelist']
 
           if 'RunWhitelist' in params: #if params has key we remove it
               params.pop('RunWhitelist') #because it was set as Task parameter
           if 'InputDataset' in params:
               params.pop('InputDataset')
+
+          if params.has_key("LumiList") and params['LumiList']:
+              params['Task1']['LumiList'] = params['LumiList']
```

Comment:
This line is only executed subject to the condition of being the the TaskChain case, see: 

https://github.com/cms-AlCaDB/wmcontrol/blob/6cbb5a4d85c37ced0138a1f65936486c2935ba1d/wmcontrol.py#L462

Can you explain better what you see the problem is ?